server/src/main/java/com/distkv/server/storeserver/runtime/expire/ExpireClient.java

```diff
@@ -0,0 +1,35 @@
+package com.distkv.server.storeserver.runtime.expire;
+
+public interface ExpireClient {
```

Comment:
Since this is a temp client, no need use interface here, just one client `ExpireClient` instead.